simulators/README.md

```diff
@@ -52,24 +100,57 @@ Detailed feature list:
         "frequency": 6
     }
   ```
-- the kind of measurement that should be sent, can be defined by
-    ```
-    "type": "PumpPressure",
-    "fragment": "Pressure",
-    "series": "P",
-    ```
-  where "type" is optional and its default value is the value from the "fragment" property
-- Simulates shutdowns (no events or measurements are sent if machine is DOWN)
-- Written in Python and is easy to extend
+  - the kind of measurement that should be sent, can be defined by
+      ```
+      "measurements": [
+          "type": "PumpPressure",
+          "fragment": "Pressure",
+          "series": "P"
+      ]
+      ```
+    where "type" is optional and its default value is the value from the "fragment" property.
+  
+  - In measurements, `valueDistribution` is defined to let the simulator know which distribution formular to use to generate measurements. There are three choices that can be defined here: `uniform`, `uniformint`, `normal`.
+
+- Simulates shutdowns (no events or measurements are sent if simulator is DOWN or out of shift)
+
 - the main entry point is [simulator.py](main/simulator.py)
   - the script reads the configuration from [simulator.json](main/simulator.json) and creates a new device for every entry
   - the `id` property is used as `external_id` for the ManagedObjects to avoid creating multiple devices when redeploying/updating the microservice
+  
 - Simulators act according to given Shiftplans
   - Simulators are linked to shiftplans via locationId
-  - Shiftplans are polled once a day
-  - If a Simulator is not in Production time according to the given Shiftplan, it will not produce any  events
-  - At startup [shiftplans.json](main/shiftplans.json) is parsed and used to update the given timeslots whitin the shiftplan
-  - The `locationId's` presented in [shiftplans.json](main/shiftplans.json) are used to parse for all shiftplans used in the script and are the only shiftplans are considered for the polling
+  - If a Simulator is not in Production time according to the given Shiftplan, it will not produce any events and measurements.
+  - At startup [shiftplans.json](main/shiftplans.json) is parsed and used to update the given timeslots within the shiftplan.
+  - The `locationId's` presented in [shiftplans.json](main/shiftplans.json) are used to parse for all shiftplans used in the script but only a specific shiftplan is applied to a simulator if it define the `locationId` of a shift.
```

Comment:
I think we need to change that a little bit:
- we need to mention that the shiftplans.json is parsed once after startup and the shifts are created accordingly (if they dont exist yet)
- everytime an event/measurement is about to be sent, the script checks if the status of the locationId == PRODUCTION; and only if the status is correct, the event/measurement is sent
- so all changes to the shiftplan are taken into account